defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+fail2ban_bantime: 3600
+fail2ban_findtime: 600
+fail2ban_maxretry: 3
+fail2ban_enabled: "false"
```

Comment:
Please use a boolean
Either jinja directly uses true/false when specifying it as boolean, otherwise you can add an if in the template to check the boolean value and write the string accordingly